kedro/io/data_catalog.py

```diff
@@ -688,8 +538,8 @@ def add_all(
             >>>
             >>> assert catalog.list() == ["cars", "planes", "boats"]
         """
-        for name, dataset in datasets.items():
-            self.add(name, dataset, replace)
+        for ds_name, ds in datasets.items():
```

Comment:
I prefer slightly longer explicit names to enhance readability, so `dataset` instead of `ds`

Reply:
There was a mix of long and short names, so I tried to follow the short option everywhere but I don not insist on it.

There are some other parts with short naming: https://github.com/kedro-org/kedro/blob/66e5e074b2789469550370f370c8b486f638d975/kedro/runner/runner.py#L102

Do we want to keep long naming everywhere?

Reply:
I personally prefer long naming, but since it's preference I won't insist either :)